src/raw_nspanel.rs

```diff
@@ -163,6 +213,23 @@ impl RawNSPanel {
         let _: () = unsafe { msg_send![self, setHidesOnDeactivate: value] };
     }
 
+    pub fn activate(&self) {
```

Comment:
Merge the contents of `activate` into the `from_window` function. In other words, perform the setup within the `from_window` method.